backend/cellguide/pipeline/computational_marker_genes/utils.py

```diff
@@ -26,13 +25,13 @@ def nanpercentile_2d(arr: np.ndarray, percentile: float, axis: int) -> np.ndarra
     """
     if axis == 0:
         result = np.empty(arr.shape[1])
-        for i in prange(arr.shape[1]):
+        for i in range(arr.shape[1]):
```

Comment:
numba does not support nested parallelism out of the box and the `tbb` threading layer (which does) is canonically finicky to install